GrandCommonDivisor/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using GrandCommonDivisor.Core;
+using GrandCommonDivisor.Core.models;
+
+namespace GrandCommonDivisor.ConsoleUI
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var first = CheckNumber("Hello User!, Please provide the first number.");
+            var second = CheckNumber("Please provide the second number.");
+            Console.WriteLine($"Here are your numbers: first: {first}, second: {second} .");
+            var result = GCDUtils.CalculateGCD(first, second);
+            Console.WriteLine($"The GCD for {first} and {second} is {result}");
+        }
+
+        private static int CheckNumber(string text)
```

Comment:
Still not the best naming. Ask yourself, what does this method do and what it returns, and based on that, create a method name.

Reply:
Fixed.